README.md.old

```diff
@@ -41,7 +55,58 @@ Checkout our [examples](./examples/README.md) directory for complete examples of
 Here is a quickstart you can use in your own project:
 
 ```typescript
-{{ usageExampleCode }}
+import {
+  CacheGet,
+  CreateCache,
+  CacheSet,
+  CacheClient,
+  Configurations,
+  CredentialProvider,
+} from '@gomomento/sdk';
+
+async function main() {
+  const momento = new CacheClient({
+    configuration: Configurations.Laptop.v1(),
+    credentialProvider: CredentialProvider.fromEnvironmentVariable({
+      environmentVariableName: 'MOMENTO_AUTH_TOKEN',
+    }),
+    defaultTtlSeconds: 60,
```

Comment:
Since this is a complete example, it's probably worth putting a comment on this to indicate to readers what it's doing. Not 100% necessary, but a suggestion

Reply:
I'd also update the example code to rename from `momento` to `cacheClient`

Reply:
this file is going to go away soon and won't be easily visible to readers after this PR is merged.

Reply:
fixed in https://github.com/momentohq/client-sdk-javascript/pull/529/commits/7772696a548e62a8805d752f9f95e5e6ac993644